django-verdant/rcasitemaps/views.py

```diff
@@ -7,6 +7,7 @@
 
 def index(request, sitemaps, **kwargs):
     sitemaps = prepare_sitemaps(request, sitemaps)
+    request.META['HTTP_HOST'] = request.site.hostname
```

Comment:
I basically debugged the request here to intercept the host value

Reply:
Result!

Might be worth checking what happens when its behind Cloudflare (or whatever CDN) as it may set `USE_X_FORWARDED_HOST` https://github.com/django/django/blob/1.11.12/django/http/request.py#L82

Reply:
Good point, from what I see we have no setting in `base.py` or `prod.py` for `USE_X_FORWARDED_HOST`

Reply:
It defaults to `False` so it should be OK. Might be worth adding a comment about why this "workaround" was added